source/designers-guide/configuration-using-theme-php/index.md

```diff
@@ -251,6 +251,31 @@ Now you can either override the default unique names like e.g. `brand-primary` o
 Every existing field which is not overwritten in your <code>Theme.php</code> will use the default value of your parent theme.
 </div>
 
+## Create a theme info tab
+
+In order to create a new tab containing information about your theme, you create a new directory in your theme folder called **info**. In that directory you can add HTML files named like the language ISO you support.
```

Comment:
In order to create a new tab containing information about your theme, create a new sub-directory in your theme directory called **info**. Add HTML files named like the language ISO you want support.